package.json

```diff
@@ -17,12 +17,13 @@
   "author": "GitHub",
   "license": "ISC",
   "dependencies": {
-    "@actions/core": "^1.2.2",
-    "@actions/github": "^2.1.0",
     "tree-kill": "^1.2.2"
   },
   "devDependencies": {
+    "@actions/core": "^1.2.2",
+    "@actions/github": "^2.1.1",
```

Comment:
Wanted to double-check that we intentionally moved this to devDependencies. I know we did it during the dev process...not sure if we need to move it back for production or if this is where it ultimately belongs.

Reply:
Yeah this is a good call out. I think that because of how we build it doesn't matter. That said I kinda don't like the inconsistency with tree-kill. I might move these back or move tree-kill down.